### ISSUE TITLE
Support for extended label on garmin maps. (extendedlabel on typ files)

### ISSUE THREAD

**Issue by salzmannii** [Maproom/qmapshack#295]
**Important Notice:**

**Do not delete any of the sections. Read what is requested.**
**_Replace(!)_ the instructions by your _answer(!)_. Feature requests**
**reports that are not properly filed might be closed without**
**further notice.**

**Support for extended label o garmin IMG maps or typ files**

[X] Did you choose a good title?

**Is your feature request related to a problem? Please describe.**
No, it's just a new feature that makes maps look visually more appealing

Extended labels are displayed like normal label. It'd make maps look visually more appealing if they were supported.

**Describe the solution you'd like**

It'd make maps look visually more appealing if they were supported.

**Describe alternatives you've considered**

**Give a reason why someone else should spend time on that request**

Extended labels in typ files include bigger and smaller fonts, colored fonts, etcetera. It makes maps easier to read.

* The request is of general interest for many users
The request would be useful for everyone who uses garmin maps. But it is not something esential.

**Additional context**

I add as an example two different views of the same map, the first one is displayed with basecamp and the other with qmapshack
![imagen](https://user-images.githubusercontent.com/77118086/103925556-0632f580-5118-11eb-9ca7-c730f295124c.png)
![imagen](https://user-images.githubusercontent.com/77118086/103926071-b7d22680-5118-11eb-87d2-db99ae833060.png)

thank you!

**Comment by kiozen** [comment]
I added the volunteer tag however I doubt that this request will find one? Why? Because it's just another one of these feature requests that places a 5 minute poo into an issue tracker asking for several man days of work. Without providing that minimum effort of pre-work that  could be done. 

No one, really no one, would like to start that job without a reference to the coding of the type file you requested and without a most simple reference file. And that is a job you do not need to know c++ for nor have to find your way into the code. It is just that chunk of work anyone could do and it would show some respect for the work and time requested to be spent.

Sorry if that sounds a bit harsh but this attitude is getting harder and harder to swallow over the years. If you want someone else to do the job for free provide as much information and pre-work as you can. You would do the same if you would ask a craftsman friend for free help in real life.

**Comment by salzmannii** [comment]
You're right, I agree with you. I am going to try to understand the typ file and add the information to this request. I guess it shouldn't be too difficult since it's not compiled in NT. I know the txt code before compiling it and I hope it is quite similar but in binary.
